src/__tests__/components/DatasetPreviewCard.test.tsx

```diff
@@ -1,24 +1,24 @@
-import { mount } from 'enzyme'
+import { shallow, mount } from 'enzyme'
 import * as React from 'react'
 import Typography from '@material-ui/core/Typography'
-import fakeTagToBadge from '../../config/__mocks__/tag-to-badge'
+// import tagToBadge from '../../config/tag-to-badge'
```

Comment:
I'm think we should maybe keep the mock config tagToBadge...let's discuss in person